Global/Eclipse.gitignore

```diff
@@ -49,3 +49,15 @@ local.properties
 
 # Code Recommenders
 .recommenders/
+
+# Java class files
+**/*.class
```

Comment:
This rule is captured by our [`Java.gitignore`](https://github.com/github/gitignore/blob/master/Java.gitignore#L1) - the expectation here is that users will grab whatever templates they need based on their usage, so I'd like to avoid duplication wherever possible.